client/src/components/Form/Elements/FormSelect.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+    <div>
```

Comment:
Add a conditional here if the is_workflow is true, it should redirect to a _fieldText type.

``` 
        if (input_def.is_workflow) {
            return this._fieldText(input_def);
        }
``` 
^ from `parameters.js`

for now, just direct it back to FormParameter, we can change this when we vueify _fieldText

```
<FormParameter
            v-if="attributes.is_workflow"
            :id="id"
            ref="params"
            v-model="currentValue"
            :data-label="title"
            :type="type"
            :attributes="attributes" />
```

Reply:
Dependency on PR #13375 - these PRs need to go out together (ideally, merged together beforehand)